src/plugins/plot/legend/PlotLegendItemCollapsed.vue

```diff
@@ -83,7 +84,8 @@ export default {
             formattedYValue: '',
             formattedXValue: '',
             mctLimitStateClass: '',
-            formattedYValueFromStats: ''
+            formattedYValueFromStats: '',
+            colorObserver: null
```

Comment:
colorObserver doesn't need to be defined here. Let's remove it.
Also, anything within the data's return {} becomes reactive, so unless it is being used by a computed property OR by the template above, we should avoid adding variables here.

Reply:
Removed! Thanks for reviewing!